### PR TITLE
Fix start and stop node scripts

### PR DESCRIPTION
**Pull request by arshdeeptinna**
Add trap to handle SIGINT and SIGTERM.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: bin/start-node [hunk 1]
```diff
@@ -11,6 +11,12 @@ set -e
 
 readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+trap cleanup INT TERM
```
Comment (lpassos) [inline bin/start-node]: Question: shouldn't it be `SIGTERM` `SIGINT` here?

File: bin/start-node [hunk 2]
```diff
@@ -11,6 +11,12 @@ set -e
...
+
+function cleanup() {
+	"$SCRIPT_DIR/stop-node"
```
Comment (lpassos) [inline bin/start-node]: What happens if the node does not stop here?

File: bin/start-node (hunk 1 above)
Comment (lpassos) [inline bin/start-node]: How is the node handling other signals---e.g., CTRL+Z? Did you test beyond CTRL+C?

Comment (arshdeeptinna) [reply bin/start-node]: changed to SIGINT and SIGTERM

File: bin/start-node (hunk 2 above)
Comment (arshdeeptinna) [reply bin/start-node]: now it should prompt user to try to forcefully stop

File: bin/start-node (hunk 1 above)
Comment (arshdeeptinna) [reply bin/start-node]: for now it does not handle other signals

**Comment by lpassos** [comment]
Thanks for the changes

**Pull request merged**